includes/commerce_pos.transaction.inc

```diff
@@ -935,3 +956,85 @@ function _commerce_pos_override_confirm_validate($form, &$form_state) {
   // Add the product to the overridden items list.
   $form_state['out_of_stock_overridden_products'][$form_state['triggering_element']['#product_id']] = $form_state['triggering_element']['#product_id'];
 }
+
+/**
+ * Returns a modal form displaying each item in an order.
+ *
+ * This popup shows a checkbox next to each item for the user to add that item
+ * to the return transaction.
+ *
+ * @param $order
```

Comment:
This is missing parameter type. 

@param object $order

That should get travis to start passing this.